cmd/cli/plugin/package/main.go

```diff
@@ -52,3 +54,22 @@ func getOutputFormat() string {
 	}
 	return format
 }
+
+func isCarvelAPIAvailable(kubeCfgPath string) (bool, error) {
```

Comment:
Isn't ```isPackagingAPIAvailable``` a slightly better naming?

Reply:
thanks for the naming advice @maralavi 
updated the function name accordingly